src/proofPoints.ts

```diff
@@ -243,6 +418,63 @@ class ProofPointsRepo {
         return proofPointObject;
     }
 
+    /**
+     * Get a list of the hashes of all proof points ever issued or committed
+     * to this registry
+     */
+    async getAll(): Promise<Array<string>> {
+        const publishEvents = await this
+            ._registry
+            .getPastEvents(
+                "Published", 
+                { 
+                    fromBlock: 0, 
+                    toBlock: "latest"
+                }
+            );
+
+        return publishEvents.map(ev => Web3.utils.hexToAscii(ev.returnValues._claim));
+    }
+
+    /**
+     * Gets a list of all events related to the given proof point, identified by its hash.
+     * @param proofPointHash the identifying hash of the proof point
```

Comment:
Clarify how is the hash generated?

Reply:
I've renamed hash to ID throughout and documented it in the spec